remote-source/go.mod

```diff
@@ -0,0 +1,74 @@
+module zephyr-automation
+
+go 1.19
```

Comment:
`go 1.19` is no longer supported[^1]. Why not use the latest: `go 1.21`?


[^1]: https://endoflife.date/go